IPIP/0342-content-router-discovery.md

```diff
@@ -0,0 +1,267 @@
+# IPIP 0342: Content Router Ambient Discovery
```

Comment:
Since I can't expose a partial index without being punished, would this be more correct?

```suggestion
# IPIP 0342: IPNI Content Router Ambient Discovery
```